docs/reference/installation.rst

```diff
@@ -42,36 +41,11 @@ are not already enabled::
     return [
         // ...
         Sonata\AdminBundle\SonataAdminBundle::class => ['all' => true],
-        Sonata\CoreBundle\SonataCoreBundle::class => ['all' => true],
         Sonata\BlockBundle\SonataBlockBundle::class => ['all' => true],
-        Sonata\EasyExtendsBundle\SonataEasyExtendsBundle::class => ['all' => true],
         FOS\UserBundle\FOSUserBundle::class => ['all' => true],
         Sonata\UserBundle\SonataUserBundle::class => ['all' => true],
     ];
 
-.. note::
-
-    If you are not using Symfony Flex, you should enable bundles in your
-    ``AppKernel.php``.
-
-.. code-block:: php
-
-    // app/AppKernel.php
-
-    public function registerBundles()
-    {
-        return [
-            new Sonata\AdminBundle\SonataAdminBundle(),
-            new Sonata\CoreBundle\SonataCoreBundle(),
-            new Sonata\BlockBundle\SonataBlockBundle(),
-            new Sonata\EasyExtendsBundle\SonataEasyExtendsBundle(),
-            // ...
-            new FOS\UserBundle\FOSUserBundle(),
-            new Sonata\UserBundle\SonataUserBundle(),
```

Comment:
Why did you remove this?

Reply:
It is still above on the docs, the recommend way on sf 4.4 is the config/bundles.php